xbmc/LangInfo.h

```diff
@@ -237,6 +239,7 @@ class CLangInfo : public ISettingCallback, public ISettingsHandler
   CRegion* m_currentRegion; // points to the current region
   CRegion m_defaultRegion; // default, will be used if no region available via langinfo.xml
   std::locale m_systemLocale;     // current locale, matching GUI settings
+  std::locale m_originalLocale; // original locale, without changes of collate
```

Comment:
Right now we take the locale for the language and we change the collate to keep constant the collate so for example atof works with '1.1' even if the language is french or italian where we use '1,1'.
The original is the locale without the changes..
It's probably not a great description.. If you have a better one suggestions are very welcome :)